templates/deployment-sidekiq.yaml

```diff
@@ -206,6 +206,24 @@ spec:
             - name: OTEL_SERVICE_NAME_SEPARATOR
               value: "{{ coalesce $context.Values.mastodon.sidekiq.otel.nameSeparator $context.Values.mastodon.otel.nameSeparator }}"
             {{- end }}
+            {{- if $context.Values.mastodon.metrics.prometheus.enabled }}
+            - name: MASTODON_PROMETHEUS_EXPORTER_ENABLED
+              value: "true"
+            - name: PROMETHEUS_EXPORTER_HOST
+              value: {{ $context.Values.mastodon.metrics.prometheus.host }}
+            - name: PROMETHEUS_EXPORTER_PORT
+              value: "{{ $context.Values.mastodon.metrics.prometheus.port }}"
+            - name: MASTODON_PROMETHEUS_EXPORTER_LOCAL
+              value: "true"
+            - name: MASTODON_PROMETHEUS_EXPORTER_HOST
```

Comment:
I dont think there is a use for this to be a variable. This is the hostname/IP the exporter listens to, not one it connects to. So for a k8s pod, is should always be `0.0.0.0`, so the exporter can be contacted from the outside?

Reply:
That's possible, though there could be a situation where someone sets up something like a sidecar for security or networking reasons, hence it being a variable. But I can remove it if you think that's mostly unnecessary.